### ISSUE TITLE
Wrong permissions in NPMJS repository archive

### ISSUE THREAD

**Issue by fpracht** [xz64/license-webpack-plugin#86]
Hi, I found a somewhat strange thing in the archive which is listed in the NPMJS repository.

I have to unpack the NPM package for an embedded project and I noticed that the permissions for the directories inside the archive are not set correctly.

Since version 2.1.3 it seems that the process has changed a bit. In version 2.1.2 everything looks correct.

>$ tar -tvzf ~/Downloads/license-webpack-plugin-2.1.2.tgz 
>drwxr-xr-x 0/0               0 2019-08-12 02:31 package
>-rw-r--r-- 0/0            7957 2019-02-23 19:26 package/CHANGELOG.md
>-rw-r--r-- 0/0            7728 2018-09-23 18:33 package/DOCUMENTATION.md
>-rw-r--r-- 0/0             742 2017-08-05 16:58 package/LICENSE
>-rw-r--r-- 0/0            2434 2018-09-23 18:33 package/MIGRATION.md
>-rw-r--r-- 0/0            1329 2018-09-23 18:33 package/README.md
>drwxr-xr-x 0/0               0 2019-08-12 02:31 package/dist
>-rw-r--r-- 0/0            1731 2019-08-12 02:29 package/package.json
>...

>$ tar -tvzf ~/Downloads/license-webpack-plugin-2.3.1.tgz 
>drw-rw-rw- 0/0               0 2020-10-24 15:50 package
>-rw-rw-rw- 0/0            8718 2020-10-24 15:32 package/CHANGELOG.md
>drw-rw-rw- 0/0               0 2020-10-24 15:50 package/dist
>-rw-rw-rw- 0/0            7937 2020-05-03 16:42 package/DOCUMENTATION.md
>-rw-rw-rw- 0/0             742 2020-03-13 15:40 package/LICENSE
>-rw-rw-rw- 0/0            2434 2020-03-13 15:40 package/MIGRATION.md
>-rw-rw-rw- 0/0            1770 2020-10-24 15:48 package/package.json
>-rw-rw-rw- 0/0            1329 2020-03-13 15:40 package/README.md
>...

The main difference is that the directories _package/_ and _package/dist/_ lack the e**x**ecute bit.

You can reproduce the whole thing by downloading the archives directly:
[https://registry.npmjs.org/license-webpack-plugin/-/license-webpack-plugin-2.3.1.tgz](https://registry.npmjs.org/license-webpack-plugin/-/license-webpack-plugin-2.3.1.tgz)
[https://registry.npmjs.org/license-webpack-plugin/-/license-webpack-plugin-2.1.2.tgz](https://registry.npmjs.org/license-webpack-plugin/-/license-webpack-plugin-2.1.2.tgz)

**Comment by xz64** [comment]
At some point I published from Windows instead of Linux. I didn't expect to see this kind of issue. I tried with the latest version of npm in Windows, and it has the same issue. I guess now is a good time to use WSL2 😄 .
I will publish a new version with corrected permissions after I set up WSL2

**Comment by xz64** [comment]
Fixed in v2.3.2

```
> tar tvf license-webpack-plugin-2.3.2.tgz
drwxr-xr-x 0/0               0 2020-12-07 20:27 package
-rw-r--r-- 0/0            8839 2020-12-07 20:25 package/CHANGELOG.md
-rw-r--r-- 0/0            7937 2020-12-07 20:11 package/DOCUMENTATION.md
-rw-r--r-- 0/0             742 2020-12-07 20:11 package/LICENSE
-rw-r--r-- 0/0            2434 2020-12-07 20:11 package/MIGRATION.md
-rw-r--r-- 0/0            1329 2020-12-07 20:11 package/README.md
drwxr-xr-x 0/0               0 2020-12-07 20:27 package/dist
-rw-r--r-- 0/0            1770 2020-12-07 20:26 package/package.json
...
```

**Comment by fpracht** [comment]
Thanks. Works for me.
